broker-cli/commands/wallet.js

```diff
@@ -241,6 +241,38 @@ const NETWORK_STATUSES = Object.freeze({
   INACTIVE: 'INACTIVE'
 })
 
+/**
```

Comment:
Moved this function above the point where it is called and added in a check for `if (!balance)`